instance/instance.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/cozy/cozy-stack/couchdb"
 	"github.com/cozy/cozy-stack/couchdb/mango"
 	"github.com/cozy/cozy-stack/vfs"
+	"github.com/dchest/uniuri"
+	scrypt "github.com/elithrar/simple-scrypt"
```

Comment:
we should use [scrypt package](https://godoc.org/golang.org/x/crypto/scrypt) from more official golang/x/crypto